shell/platform/windows/window_win32.cc

```diff
@@ -167,9 +168,11 @@ void WindowWin32::OnGetObject(UINT const message,
     // Microsoft Active Accessibility (MSAA) COM objects.
     OnUpdateSemanticsEnabled(true);
 
-    // TODO(cbracken): https://github.com/flutter/flutter/issues/77838
-    // Once AccessibilityBridge is wired up, look up the IAccessible
-    // representing the root view and call LresultFromObject.
+    // Return the IAccessible for the root view.
+    gfx::NativeViewAccessible root_iaccessible = GetNativeViewAccessible();
+    if (root_iaccessible) {
+      LresultFromObject(IID_IAccessible, wparam, root_iaccessible);
+    }
```

Comment:
is there a way to actively attach the root accessibility node to the view?

The first time this method is called root_iaccessible will be null because the update will only come after one frame delay.

Reply:
btw, Is this also how chromium listens to the accessibility services?

Reply:
This is how the root accessibility node is attached to the view.

When Windows wants the accessibility node associated with the HWND it fires a `WM_GETOBJECT` event into the main event loop, which we then handle in this method. We look up the root `AXPlatformNodeWin` (which implements `IAccessible`) above in `GetNativeViewAccessible()` and return it via `LresultFromObject()` here.